tests/logic/office_test.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
```

Comment:
Could you follow up here and add tests for `get_office_name`?

Reply:
Sure I can do that